src/app/_hooks/useE2EE.ts

```
@@ -0,0 +1,189 @@
+"use client";
+
+import { useCallback, useEffect, useRef, useState } from "react";
+import { useSession } from "next-auth/react";
```

ESLint

[error] `next-auth/react` import should occur before import of `react`

(import/order)